src/app/settings/settings.reducer.spec.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  initialState,
+  settingsReducer,
+  ActionSettingsPersist,
+  SettingsActionTypes,
+  ActionSettingsChangeTheme,
+  ActionSettingsChangeAnimationsPage,
+  ActionSettingsChangeAnimationsPageDisabled,
+  ActionSettingsChangeAnimationsElements,
+  ActionSettingsChangeAutoNightMode
+} from './settings.reducer';
+
+describe('settingsReducer', () => {
+  it('ActionSettingsPersist with undefined new state should return the initial state', () => {
+    const action: ActionSettingsPersist = {
```

Comment:
This action doesn't have reducer handler so its basically as testing `reducer(undefined, anythingRandom)` which would also trigger `default` reducer handler. As so I guess this test can be replaced with

```typescript
it('should return the default state', () => {
    const action = {} as any;
    const state = todosReducer(undefined, action);

    expect(state).toBe(initialState);
  });
```

because it's not specific to that persist action.